.travis.yml

```diff
@@ -23,16 +23,15 @@ services:
   - redis-server
 
 before_script:
-  - docker-compose up -d dynamodb
+  - docker-compose up -d localstack
   - mysql -e 'CREATE DATABASE test;'
   - psql -c 'CREATE DATABASE test;' -U postgres
 
-  # waiting for DynamoDB
-  - while [[ $(curl -so /dev/null -w '%{response_code}' localhost:8000) != '400' ]]; do sleep 1; done
+  # waiting for the DynamoDB mock
+  - while [[ $(curl -so /dev/null -w '%{response_code}' localhost:4569) != '502' ]]; do sleep 1; done
```

Comment:
nit: for better visibility, `do sleep 1; done` => `do echo 'waiting 1sec for DynamoDB...'; sleep 1; done`

Reply:
Done.